device-posture/device-posture-change-event.https.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/resources/testdriver.js"></script>
+<script src="/resources/testdriver-vendor.js"></script>
+<script>
+'use strict';
+
+// REFERENCE https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/web_tests/http/tests/inspector-protocol/device-posture/device-posture-change-event.js
+promise_test(async (t) => {
+  const watcher = new EventWatcher(t, navigator.devicePosture, ['onchange']);
+  const postures = ['continuous', 'folded', 'continuous', 'folded'];
```

Comment:
Fails if code is `const postures = ['folded', 'continuous', 'folded'];`
Fails on first loop by saying `navigator.devicePosture.type` is `continuous` and should be `folded`.

Reply:
Fixed in https://github.com/JuhaVainio/WPT/pull/4/commits/291385c15ed5cf39ad5a37182bcc1994ee799744